internal/bminventory/inventory_test.go

```diff
@@ -2161,6 +2160,35 @@ var _ = Describe("cluster", func() {
 		})
 
 		Context("Monitored Operators", func() {
+			var (
+				testOLMOperators = []*models.MonitoredOperator{
+					{
+						Name:           "0",
+						OperatorType:   models.OperatorTypeOlm,
+						TimeoutSeconds: 1000,
+					},
+					{
+						Name:           "1",
+						OperatorType:   models.OperatorTypeOlm,
+						TimeoutSeconds: 2000,
+					},
+				}
+			)
+
+			getOperatorMock := func(operatorName string) {
```

Comment:
The function is not a mock itself - it mocks desired behavior. I think the name should reflect its purpose; like `mockGetOperatorByName` or similarly.